examples/domainmodel/package.json

```diff
@@ -49,24 +58,24 @@
         "publish:latest": "npm publish --tag latest"
     },
     "dependencies": {
-        "colors": "^1.4.0",
         "commander": "^8.0.0",
         "langium": "0.4.0",
         "lodash": "^4.17.21",
         "vscode-languageclient": "^8.0.1",
         "vscode-languageserver": "^8.0.1"
     },
     "devDependencies": {
+        "chalk": "^4.1.2",
```

Comment:
Suggestion: See the comment above.

Reply:
fixed - see commit [79e303b](https://github.com/langium/langium/pull/578/commits/79e303b850b59ada15e8fe4d9e0d7a3017386bca)